src/libs/ReportUtils.ts

```diff
@@ -6386,7 +6386,7 @@ function isReportNotFound(report: OnyxEntry<Report>): boolean {
 /**
  * Check if the report is the parent report of the currently viewed report or at least one child report has report action
  */
-function shouldHideReport(report: OnyxEntry<Report>, currentReportId: string): boolean {
+function shouldHideReport(report: OnyxEntry<Report>, currentReportId: string | undefined): boolean {
```

Comment:
Can we do an early return here when `currentReportId` is undefined?

Reply:
I don't think we should do it because the logic depends on `isChildReportHasComment`. I would prefer to just let it go through the code since the parentReport will be undefined when currentReportID is undefined.